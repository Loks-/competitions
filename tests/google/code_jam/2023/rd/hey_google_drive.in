4
1 2
aZ
4 4
a..c
**.*
.Y.#
bX#Z
2 2
a*
*Z
2 7
a*bcd*.
...*F#.
