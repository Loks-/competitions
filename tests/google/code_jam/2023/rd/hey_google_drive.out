Case #1: aZ
Case #2: aY bX bY cY
Case #3: NONE
Case #4: dF
